src/main/config/secor.common.properties

```diff
@@ -519,6 +519,10 @@ parquet.enable.dictionary=true
 # Parquet 1.9.
 parquet.validation=false
 
+# Avro doesn't use the hadoop codec for compression it includes his own (see here: org.apache.avro.file.CodecFactory)
+# value can be null(no compression),deflate,snappy,bzip2
+avro.compression.codec=null
```

Comment:
'null' as default is unconventional, 'none' might be better.

Reply:
switched to "none" string